doc/api/cli.md

```diff
@@ -525,6 +525,10 @@ malformed, but any errors are otherwise ignored.
 Note that neither the well known nor extra certificates are used when the `ca`
 options property is explicitly specified for a TLS or HTTPS client or server.
 
+`NODE_EXTRA_CA_CERTS` is not intended to be used to set the paths of extra
```

Comment:
I find the text confusing here, I'm not sure what is meant by "setting paths". Rather than rewriting though, I suggest removing the entire doc change, both sentences.  Its unnecessary, env variables are intended to be set outside node, not by node to configure itself at runtime.

Reply:
Right. My explanation here makes the doc even confusing.